tables/Tables/ArrayColumn.tcc

```diff
@@ -212,21 +212,21 @@ void ArrayColumn<T>::getSlice (uInt rownr, const Slicer& arraySection,
         //# Creating a Slicer is somewhat expensive, so use the slicer
         //# itself if it contains no undefined values.
         if (arraySection.isFixed()) {
-	    baseColPtr_p->getSlice (rownr,
-				    arraySection,
-				    &arr);
-	} else {
-	    baseColPtr_p->getSlice (rownr,
-				    Slicer(blc, trc, inc,
-					   Slicer::endIsLast),
-				    &arr);
-	}
+            baseColPtr_p->getSlice (rownr,
+                                    arraySection,
+                                    &arr);
+        } else {
+            baseColPtr_p->getSlice (rownr,
+                                    Slicer(blc, trc, inc,
+                                           Slicer::endIsLast),
+                                           &arr);
```

Comment:
That method wasn't even supposed to be modified; so I don't know how it 
ended up getting reformatted.

On 06/25/2015 12:31 AM, Ger van Diepen wrote:
>
> In tables/Tables/ArrayColumn.tcc 
> <https://github.com/casacore/casacore/pull/121#discussion_r33225777>:
>
> > -				    arraySection,
> > -				    &arr);
> > -	} else {
> > -	    baseColPtr_p->getSlice (rownr,
> > -				    Slicer(blc, trc, inc,
> > -					   Slicer::endIsLast),
> > -				    &arr);
> > -	}
> > +            baseColPtr_p->getSlice (rownr,
> > +                                    arraySection,
> > +                                    &arr);
> > +        } else {
> > +            baseColPtr_p->getSlice (rownr,
> > +                                    Slicer(blc, trc, inc,
> > +                                           Slicer::endIsLast),
> > +                                           &arr);
>
> This indentation is incorrect.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/casacore/casacore/pull/121/files#r33225777>.Web 
> Bug from 
> https://github.com/notifications/beacon/AGPGtF_Rcmznkvu6lDl6M1kFdWKMPFK-ks5oW5fIgaJpZM4FDwg4.gif
>

-- 
Jim Jacobs

phone: 835-7235
office: 301